packages/framework/src/Model/Product/ProductRepository.php

```diff
@@ -162,10 +154,12 @@ public function addTranslation(QueryBuilder $queryBuilder, $locale)
      * @param \Doctrine\ORM\QueryBuilder $queryBuilder
      * @param int $domainId
      */
-    public function addDomain(QueryBuilder $queryBuilder, $domainId)
+    private function addDomain(QueryBuilder $queryBuilder, $domainId)
```

Comment:
Generally, there shouldn't be private methods in repositories as they are supposed to be extended by inheritance.